lib/appsignal.ex

```diff
@@ -116,7 +116,9 @@ defmodule Appsignal do
 
   @doc false
   def add_default_probes do
-    Appsignal.Probes.register(:erlang, &Appsignal.Probes.ErlangProbe.call/0)
+    if Config.minutely_probes_enabled?() do
```

Comment:
We probably don't want to start the Probe Genserver in that case?

Reply:
Yes! Can you review?